client/me/purchases/manage-purchase/index.jsx

```diff
@@ -625,6 +625,7 @@ class ManagePurchase extends Component {
 			getAddPaymentMethodUrlFor,
 			getEditPaymentMethodUrlFor,
 			isProductOwner,
+			showHeader,
```

Comment:
Why not make this `showTitle` to match the others?

Reply:
nevermind

Reply:
I was an existing prop but it's gone now :)